modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,71 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using one of the following authentication methods:
+
+* One-time password generator
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+
+== Multi-factor Authentication Configurations
+
+You can configure the following verification methods to comply with multi-factor authentication requirements:
+
+One-time password generator:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID, to verify your identity when logging in to Anypoint Platform.
+
+// what happens if they lose this device, and it's their only MFA method?...
```

Comment:
We should make a call out somewhere:  "If you lose your multi-factor verification method, you should contact an org administrator, who can reset MFA for a user"

Reply:
How does the org admin reset MFA? Adding them to the exemption list?

Reply:
They go to the same area of user management used to "reset a password".  The link will be "Reset multi-factor authentication"

Reply:
Sick. I'll add it.

Reply:
Added.